maskrcnn_benchmark/structures/keypoint.py

```diff
@@ -11,9 +11,9 @@ def __init__(self, keypoints, size, mode=None):
         # in my version this would consistently return a CPU tensor
         device = keypoints.device if isinstance(keypoints, torch.Tensor) else torch.device('cpu')
         keypoints = torch.as_tensor(keypoints, dtype=torch.float32, device=device)
-        num_keypoints = keypoints.shape[0]
-        if num_keypoints:
-            keypoints = keypoints.view(num_keypoints, -1, 3)
+        num_persons = keypoints.shape[0]
```

Comment:
Before the reshaping, the first dimension of the tensor keypoints should represent the number of persons.